drivers/input/Kconfig.tsc_keys

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2024 Arif Balik <arifbalik@outlook.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config STM32_TSC
+	bool "STM32 TSC driver"
+	default y
+	depends on DT_HAS_ST_STM32_TSC_ENABLED
+	help
+	  Enable driver for STM32 TSC peripheral.
+
```

Comment:
does it make sense to still have this a separate option now? may as well fold it back in the one below

Reply:
Makes no sense